inputs/manual_input.csv

```diff
@@ -485,21 +485,24 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
 steel carbon capture retrofit,gas-input,2030,0.76,MWh/tCO2,2019,"National Petroleum Council, Meeting the Dual Challenge - A Roadmap to At-Scale Deployment of Carbon Capture, Use, and Storage: https://dualchallenge.npc.org/files/CCUS-Chap_2-030521.pdf, p2-20, accessed 2025-04-16",
 steel carbon capture retrofit,investment,2030,3536679,USD/(tCO2/h),2019,"National Petroleum Council, Meeting the Dual Challenge - A Roadmap to At-Scale Deployment of Carbon Capture, Use, and Storage: https://dualchallenge.npc.org/files/CCUS-Chap_2-030521.pdf, p2-18, accessed 2025-04-16","Capital cost 1342 million USD, CO2 Volume captured 3324000 t/year"
 steel carbon capture retrofit,lifetime,2030,20,years,2019,"National Petroleum Council, Meeting the Dual Challenge - A Roadmap to At-Scale Deployment of Carbon Capture, Use, and Storage: https://dualchallenge.npc.org/files/CCUS-Chap_2-030521.pdf, p2-21, accessed 2025-04-16",
-Alkaline electrolyzer,lifetime,2020,6,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
-Alkaline electrolyzer,lifetime,2050,9,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+Alkaline electrolyzer,lifetime,2020,40,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```

Comment:
Looking at the updated numbers, these will results in drastically different results compared to "Alkaline electrolyzer, medium size, centralized". These seem overoptimistic. Can we as a compromise, add both technologies:

"Alkaline Electrolyser, large size, centralized"
and
"Alkaline Electrolyser, medium size, centralized"